plugins/dashboard/frontend/src/app/components/ExplorerAddressResult.tsx

```diff
@@ -117,7 +117,8 @@ export class ExplorerAddressQueryResult extends React.Component<Props, any> {
                             <div>{status}</div>
                             <div>{consumed}</div>
                             <div>{conflicting}</div>
-                            <div>{'Balance:'} {balances}</div>   
+                            <div>{'Balance:'} {balances}</div>
+                            <div>Pending Mana: {output.pending_mana}</div>

```

Comment:
there is a `displayManaUnit` function in `ManaGauge`, I think we could put it into some utils or helpers dir and reuse it here too.